addons/upgrade/to-10.2-default-switch-packetfence-standard.pl

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/perl
+
+=head1 NAME
+
+to-10.2-default-switch-packetfence-standard.pl
+
+=cut
+
+=head1 DESCRIPTION
+
+Add Generic type where needed
+
+=cut
```

Comment:
If purpose of this PR is to "Set the default switch type to Packetfence::Standard", title and description needs to be adjust.

Reply:
Also description of PR (Upgrade section) need to be edited.

Reply:
Purpose of this script is to set Generic type to switch which don't have any type.